packages/client/src/utils/index.test.tsx

```diff
@@ -61,6 +62,18 @@ describe("util", () => {
     expect(isValidPostalcode(" 1055XD ")).toBe(true);
     expect(isValidPostalcode(" 1055 XD ")).toBe(true);
   });
+  test("transformHousenumberToValid", () => {
```

Comment:
We spell somthinghousenumber like `somethingHouseNumber`. Even though I'm wondering about the clarity of this function name. @afjlambert what do you think?